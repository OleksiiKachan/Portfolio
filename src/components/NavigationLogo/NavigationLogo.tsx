import React, { Component } from 'react';
import classnames from 'classnames';

/*
    <NavigationLogo />
*/
class NavigationLogo extends Component<any, any> {
  render() {
    return (
      <div className={classnames('navigation__logoWrapper')}>
        <img
          alt="Oleksii Kachan logo"
          className={classnames('navigation__logoImage')}
          src="https://res.cloudinary.com/oleksiikachan/image/upload/v1546142703/portfolio/images/logo/logo_horizontal_dark.svg"
        />
      </div>
    );
  }
}

export default NavigationLogo;
