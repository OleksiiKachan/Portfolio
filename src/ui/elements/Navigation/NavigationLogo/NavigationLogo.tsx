import React from 'react';
import classnames from 'classnames';

/*
    <NavigationLogo
      className=''
    />
*/

type PropsType = {
  className?: string;
};

export default ({ className }: PropsType) => {
  return (
    <div className={classnames('navigation__logoWrapper', className)}>
      <img
        alt="Oleksii Kachan logo"
        className={classnames('navigation__logoImage')}
        src="https://res.cloudinary.com/oleksiikachan/image/upload/v1546142703/portfolio/images/logo/logo_horizontal_dark.svg"
      />
    </div>
  );
};
