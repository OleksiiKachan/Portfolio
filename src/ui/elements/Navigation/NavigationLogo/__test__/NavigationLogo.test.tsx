import React from 'react';
import { shallow } from 'enzyme';
import NavigationLogo from '../NavigationLogo';

describe('NavigationLogo', () => {
  it('Snapshot', () => {
    const component = shallow(<NavigationLogo />);
    expect(component).toMatchSnapshot();
  });
});
