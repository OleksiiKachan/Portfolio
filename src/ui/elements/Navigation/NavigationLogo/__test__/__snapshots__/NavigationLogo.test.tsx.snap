// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationLogo Snapshot 1`] = `
<div
  className="navigation__logoWrapper"
>
  <img
    alt="Oleksii Kachan logo"
    className="navigation__logoImage"
    src="https://res.cloudinary.com/oleksiikachan/image/upload/v1546142703/portfolio/images/logo/logo_horizontal_dark.svg"
  />
</div>
`;
